.codeclimate.yml

```diff
@@ -2,4 +2,4 @@ version: "2"
 checks:
   argument-count:
     config:
-      threshold: 6
+      threshold: 8
```

Comment:
That's because of `runInternal` that takes in all the dependencies as arguments. The fix to reduce the argument count would be to wrap all the dependencies in a `struct` and pass that but I think I'll just do that next time. And yes, it's awkward because I'm passing in the `slack.Client` for many arguments because that's what's implementing many of the interfaces that I need but that's still the right way to do it (by that, I mean using the small interfaces instead of giving up and having a single argument of type `*slack.Client`).

Reply:
Funny thing is that writing what I should do later made me want to just take care of it so it's ✅ .